.gitignore

```diff
@@ -1,7 +1,9 @@
 composer.phar
 composer.lock
 vendor
+tools/php-cs-fixer/vendor
```

Comment:
`.gitattributes` should probably not export this folder at all

Reply:
Good point, fixed